src/Workspaces/Core/MSBuild/MSBuild/MSBuildProjectLoader.Worker.cs

```diff
@@ -384,7 +384,8 @@ private Task<ProjectInfo> CreateProjectInfoAsync(ProjectFileInfo projectFileInfo
                         analyzerReferences: analyzerReferences,
                         additionalDocuments: additionalDocuments,
                         isSubmission: false,
-                        hostObjectType: null);
+                        hostObjectType: null)
+                        .WithDefaultNamespace(projectFileInfo.DefaultNamespace);
```

Comment:
why is the DefaultNamespcae not passed in as an argument?

Reply:
It was using the public constructor, I don't want to change that.

Reply:
sorry, it just feels totally out of place.  There should be a constructor that takes this arg.  And the pattern here is to just construct, not construct then mutate and mutate etc.  We don't do this with any of hte other args, so i don't see why we'd do with with 'DefaultNamespace'...

Reply:
Not fixing this to keep it internal. Tracked by https://github.com/dotnet/roslyn/issues/30932